yb-voyager/cmd/analyzeSchema.go

```diff
@@ -612,21 +610,18 @@ func checkPlPgSQLStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType st
 			continue
 		}
 		for _, issueInstance := range issues {
-			issue := convertIssueInstanceToAnalyzeIssue(issueInstance, fpath, true)
+			issue := convertIssueInstanceToAnalyzeIssue(issueInstance, fpath, true, true)
 			schemaAnalysisReport.Issues = append(schemaAnalysisReport.Issues, issue)
 		}
 	}
 
 }
 
-func convertIssueInstanceToAnalyzeIssue(issueInstance queryissue.QueryIssue, fileName string, isPlPgSQLIssue bool) utils.AnalyzeSchemaIssue {
+func convertIssueInstanceToAnalyzeIssue(issueInstance queryissue.QueryIssue, fileName string, isPlPgSQLIssue bool, addToSummaryMap bool) utils.AnalyzeSchemaIssue {
```

Comment:
nit: few comments about this func args will help. 
Like when/why we need/don't-need the addToSummaryMap

Q. Can we not always add to summary map? I see its just store invalid count info in a set

Reply:
That summary map is used in the analyze schema part for TABLES, TRIGGERS etc to store summary related issues afair. We have no need of that in case of assessment report in datatypes. I have added a relevant comment there.